### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/04/day3.html?fbclid=IwAR0FECKX8jzGLCMQgTa601XmoQBzhLM05CNQPriZwo9mioAXSnsrBDULiVg_aem_Af8JH9Uob-X7ZtXLPLmXHbykFdjzxbBbp5HrtqCZu417K9H7mKFzCPjlluHpgS9-6sU

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#9] [Comments]
# https://mnfienen.github.io/talulat/2023/08/04/day3.html?fbclid=IwAR0FECKX8jzGLCMQgTa601XmoQBzhLM05CNQPriZwo9mioAXSnsrBDULiVg_aem_Af8JH9Uob-X7ZtXLPLmXHbykFdjzxbBbp5HrtqCZu417K9H7mKFzCPjlluHpgS9-6sU

Day 3 - Beautiful Teamwork miles today: 93 total miles: 169 km today: 150 total km: 192

https://mnfienen.github.io/talulat/2023/08/04/day3.html?fbclid=IwAR0FECKX8jzGLCMQgTa601XmoQBzhLM05CNQPriZwo9mioAXSnsrBDULiVg_aem_Af8JH9Uob-X7ZtXLPLmXHbykFdjzxbBbp5HrtqCZu417K9H7mKFzCPjlluHpgS9-6sU

**Comment by snoopydance07** [comment]
hi Chandra!!  dayumm, that staircase!?!?!  what a gut-puncher!  beautiful teamwork indeed!  i’m glad you replaced the wheel!

**Comment by aleaf** [comment]
You just never know… that’s cool that you  had the 40 spoke rim in front.

**Reply by mnfienen** [reply]
yeah @aleaf - Santana def made a pro design move on that one!
